app/controllers/users_controller.rb

```diff
@@ -1,5 +1,12 @@
 class UsersController < ApplicationController
-  before_action :load_user, only: %i(show edit)
+  before_action :logged_in_user, only: %i(index edit update)
```

Comment:
check phải login thì mới được thao tác với action show và destroy nữa nha em. Nên chuyển qua except cho code gọn hơn
```suggestion
  before_action :logged_in_user, except: %i(new create)
```